### PR TITLE
Added Dell Inspiron 15 7579

### PR DESCRIPTION
**Pull request by Schyken**
I've added my current laptop, the Dell Inspiron 15 7579, to the list as it functions as intended with Solus installed. There have been no graphical issues, threading delays, UEFI errors, or TPM failures.

### CODE REVIEW

File: hardware/laptops/en.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ below that are in fact compatible.
 - Dell Inspiron 15R N5110
 - Dell Inspiron 5050
 - Dell Inspiron 7548
+- Dell Inspiron 15 7579
```
Comment (JoshStrobl) [inline hardware/laptops/en.md]: This should go before `Dell Inspiron 15R N5010`

**Comment by Schyken** [comment]
I believe I've corrected it (apologies, as I'm not very skilled with GitHub).

**Comment by JoshStrobl** [comment]
This should be a squashed commit :P

**Comment by JoshStrobl** [comment]
Actually nevermind, I'll just merge it as-is.

**Pull request merged**